src/axom/sidre/tests/sidre_native_layout.cpp

```diff
@@ -264,6 +266,254 @@ TEST(sidre_native_layout,native_layout_with_scalars)
 }
 
 
+//------------------------------------------------------------------------------
+TEST(sidre_native_layout,export_import_conduit)
+{
+  // Checks that we can import an exported conduit tree into a separate
+  // datastore using Group::createNativeLayout() and Group::importConduitTree()
+
+  // Tests with non-zero- and with zero-sized arrays
+  for( int SZ : {10, 0})
+  {
+    SLIC_INFO("Checking export/import from conduit with array of size " << SZ);
+
+    axom::sidre::DataStore ds1, ds2;
+    conduit::Node node1, node2;
+
+    int* conduit_data = nullptr;
+
+    std::string grpName = "some_group";
+    std::string viewName = "array_view";
+
+    // Simple lambda to return the expected value at an index
+    auto expValue = [=](int i) {
```

Comment:
We define `AXOM_HOST_LAMBDA` as well, which could be used here instead (and in other places below)

Reply:
Thanks @gzagaris -- I'll update it.